.github/workflows/codespell.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       - uses: codespell-project/actions-codespell@master
         with:
           skip: ca_hashes.txt,tls_data.txt,*.pem,OPENSSL-LICENSE.txt
-          ignore_words_list: borken,gost,ciph,ba,bloc,isnt,chello,fo,alle
+          ignore_words_list: borken,gost,ciph,ba,bloc,isnt,chello,fo,alle,aNULL
```

Comment:
Change `aNULL` → `anull`

Reply:
Will try

Reply:
Ok, for 3.0 it worked, see: #2335.

My question for the future: How am I supposed to know? The complaint was ``aNULL ==> annul``.

Reply:
Thanks, btw! 👍

Reply:
Words are written with lower case letters in the dictionaries of typos, except for proper nouns (Mircosoft for example). I am not sure there is currently a better way to find than looking in the dictionaries un der [codespell_lib/data](https://github.com/codespell-project/codespell/tree/master/codespell_lib/data).

Reply:
Thanks. I did that, using aNULL. But for some reason git doesn't want to tell me: https://github.com/codespell-project/codespell/search?q=anull .

Reply:
That's a GitHub issue. Use other means to search files.

Reply:
sigh. The files are plain text files. But sure I can myself.